app/services/state_file/state_information_service.rb

```diff
@@ -101,23 +103,24 @@ def state_code_to_name_map
       nj: {
         intake_class: StateFileNjIntake,
         calculator_class: Efile::Nj::Nj1040,
+        filing_years: [2024],
         navigation_class: Navigation::StateFileNjQuestionNavigation,
         submission_builder_class: SubmissionBuilder::Ty2024::States::Nj::NjReturnXml,
         state_name: "New Jersey",
         return_type: "Resident",
         schema_file_name: "NJIndividual2023V0.4.zip",
-        optional: true
-        # mail_voucher_address: "New Jersey Personal Income Tax<br/>" \
-        #                       "Processing Center<br/>" \
-        #                       "Trenton, NJ".html_safe,
-        # pay_taxes_link: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
-        # survey_link: "",
-        # tax_payment_info_url: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
-        # tax_payment_url: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
-        # tax_refund_url: "https://www.tax.ny.gov/pit/file/refund.htm",
-        # vita_link: "",
-        # voucher_form_name: "NJ Voucher Form",
-        # voucher_path: "/pdfs/it201v_1223.pdf", # TODO
+        optional: true,
+        mail_voucher_address: "New Jersey Personal Income Tax<br/>" \
+                              "Processing Center<br/>" \
+                              "Trenton, NJ".html_safe,
+        pay_taxes_link: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
+        survey_link: "",
+        tax_payment_info_url: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
+        tax_payment_url: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
+        tax_refund_url: "https://www.tax.ny.gov/pit/file/refund.htm",
+        vita_link: "",
+        voucher_form_name: "NJ Voucher Form",
+        voucher_path: "/pdfs/it201v_1223.pdf", # TODO
```

Comment:
[DUST] Is it possible to leave voucher_path commented? Or at least blank if not? Concerned about pointing to a nonexistent path.

Reply:
Or we could set it to an empty string?

Reply:
done!